PLEP-0007.rst

```diff
@@ -0,0 +1,262 @@
+|   The focus of an analysis routine should be made as broad as possible.      |
+|   When the routine's functionality becomes highly-tailored to its            |
+|   application, then it should be placed into an explicitly appropriate       |
+|   namespace.  For example, linear, langmuir, magnetic-flux, etc. analysis    |
+|   tools should be placed in their respective namespaces.                     |
++------------------------------------------------------------------------------+
+| .. _diagnostics:                                                             |
+|                                                                              |
+| **diagnostics**                                                              |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/diagnostics/``                                      |
+| :access: ``plasmapy.diagnostics``                                            |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package that fully defines the parameters of a "diagnostic."  A      |
+|   "diagnostic" is any data collecting instrument ranging from experimental   |
+|   probes, like Langmuir and magnetic flux probes, to analogous synthetic     |
+|   diagnostics for simulations.                                               |
+|                                                                              |
+|   Tools in this package do not define any analysis routines, but focus on    |
+|   defining probe characteristics/calibrations that can be seamlessly passed  |
+|   to ``plasmapy.analysis`` routines.                                         |
+|                                                                              |
+|   A "``Diagnostic``" class should fully define its diagnostic                |
+|   characteristics and provide access to its most relevant analysis tools     |
+|   found in ``plasmapy.analysis``.                                            |
++------------------------------------------------------------------------------+
+| .. _dispersion:                                                              |
+|                                                                              |
+| **dispersion**                                                               |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/dispersion/``                                       |
+| :access: ``plasmapy.dispersion``                                             |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package dedicated to solving various plasma dispersion relations.    |
```

Comment:
Maybe `...solving dispersion relations for various plasma waves and instabilities.`?

Reply:
Maybe something more like...

> A sub-package dedicated to providing tools to work with plasma dispersion relations.  This includes, but not limited to, solving various dispersion relations, defining dispersion functions, plotting dispersion relations, etc.